litani

```diff
@@ -281,9 +172,39 @@ def get_args():
         flags = arg.pop("flags")
         graph_pars.add_argument(*flags, **arg)
 
+    get_jobs_pars = subs.add_parser("get-jobs", help="print jobs")
+    for arg in [{
+        "flags": ["-f", "--out-file"],
+        "help": "write the list of jobs to a JSON file, default: stdout",
+        "default": sys.stdout,
+        "metavar": "F",
+        "type": argparse.FileType("w")
+    }]:
+        flags = arg.pop("flags")
+        get_jobs_pars.add_argument(*flags, **arg)
+    get_jobs_pars.set_defaults(func=lib.jobs.print_jobs)
+
+    set_jobs_pars = subs.add_parser("set-jobs",
+        help="set jobs to run")
+    set_jobs_mutually_exclusive = set_jobs_pars.add_mutually_exclusive_group()
+    for arg in [{
+            "flags": ["-j", "--jobs"],
+            "help": "read a list of jobs from a JSON string",
+            "metavar": "J"
+    }, {
+            "flags": ["-f", "--from-file"],
+            "help": "read a list of jobs from a JSON file, default: stdin",
+            "metavar": "F",
+            "default": sys.stdin,
+            "type": argparse.FileType("r")
+    }]:
+        flags = arg.pop("flags")
+        set_jobs_mutually_exclusive.add_argument(*flags, **arg)
+    set_jobs_pars.set_defaults(func=set_jobs)
```

Comment:
Why isn't this `lib.jobs.set_jobs`?

Reply:
Because I have a helper function in the litani file which loads the list of jobs based on how the user gave them to us and then calls the `lib.jobs.set_jobs()` function and passes in the list of jobs.